### ISSUE TITLE
Patch for /port/atomic_pointer.h

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#123]
```
Porting to HP-UX ia64:
ia64 AtomicPointer implementation

```

---

Original issue reported on code.google.com by `Alexande...@gmail.com` on 3 Oct 2012 at 7:41

Attachments:
- [atomic_pointer.h.patch](https://storage.googleapis.com/google-code-attachments/leveldb/issue-123/comment-0/atomic_pointer.h.patch)

**Comment by GoogleCodeExporter** [comment]
```
Could you take a moment to e-sign the contributor license agreement
at http://code.google.com/legal/individual-cla-v1.0.html
so that I can incorporate your patches.  Email me when you have done so.

Thanks very much.

```

---

Original comment by `san...@google.com` on 5 Oct 2012 at 7:58

**Comment by GoogleCodeExporter** [comment]
```
Patch modified: added "#undef ARCH_CPU_IA64_FAMILY" at the end of the header
```

---

Original comment by `Alexande...@gmail.com` on 9 Oct 2012 at 2:02

Attachments:
- [atomic_pointer.h.patch](https://storage.googleapis.com/google-code-attachments/leveldb/issue-123/comment-2/atomic_pointer.h.patch)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.6.0.

```

---

Original comment by `san...@google.com` on 12 Oct 2012 at 7:54
- Changed state: **Fixed**
